examples/agents_with_errors.py

```
@@ -0,0 +1,73 @@
+    with promising.context():
+        return agent3_plain_func2()
+
+
+@promising.function
+async def agent3() -> Promise[None]:
+    try:
+        return await agent3_plain_coro1()
+    except Exception as e:
+        # The error will NOT go through here - it will be raised later, when
+        # the promise is unpacked
+        raise e
```

Ruff (0.15.12)

[warning] Remove exception handler; error is immediately re-raised

(TRY203)